bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -337,6 +338,17 @@ def RDF(self):
         del type(self).RDF
         return ns
 
+    # Overload RooFit namespace
+    @property
+    def RooFit(self):
+        ns = self._fallback_getattr('RooFit')
+        try:
+            self._pythonization._roofit.pythonize_roofit_namespace(ns)
```

Comment:
What about not making `self._pythonization` a member of the facade and just do here, in this property, `from .pythonization._roofit import pythonize_roofit_namespace`? See a similar import above in this file `from ._application import PyROOTApplication`.